src/components/MultiHopTrade/components/SharedTradeInput/SharedRecipientAddress.tsx

```diff
@@ -185,6 +185,8 @@ export const SharedRecipientAddress = ({
         validateAddress: async (rawInput: string) => {
           try {
             const value = rawInput.trim() // trim leading/trailing spaces
+            // Don't go invalid on initial empty string
+            if (!value) return true
```

Comment:
Cheeky regression on the third bird here ser: empty space(s) is now a valid address 💀

<img width="427" alt="Screenshot 2025-04-01 at 3 19 51 pm" src="https://github.com/user-attachments/assets/614d464e-43ff-4991-98a6-bda9695fe1d6" />
<img width="446" alt="Screenshot 2025-04-01 at 3 19 58 pm" src="https://github.com/user-attachments/assets/5d720a51-7230-4a7c-8a08-60f3946d5df8" />

Reply:
V. nice catch ser @0xApotheosis - fixed in c3a94112e8